doc/conf.py

```diff
@@ -1,215 +1,213 @@
 # autogenerated file.
 #
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-# If your extensions are in another directory, add it here. If the directory
-# is relative to the documentation root, use os.path.abspath to make it
-# absolute, like shown here.
-# sys.path.append(os.path.abspath('.'))
-
-# General configuration
-# ---------------------
-# import Cloud
-import cloud_sptheme as csp
-
-# ... some contents omitted ...
-
-# set the html theme
-html_theme = "cloud"
-
-# ... some contents omitted ...
-
-# set the theme path to point to cloud's theme data
-html_theme_path = [csp.get_theme_dir()]
-
-# [optional] set some of the options listed above...
-html_theme_options = {
-    "sidebarwidth": "200px",
-    "max_width": "900px",
-    "compact_width": "800px",
-    "minimal_width": "700px",
-
-    # Style it like Fedora..
-    "bodyfont": "Cantarell",
-
-    "highlightcolor": "#4FAF4F",
-
-    "sidebarbgcolor": "#FEFEFE",
-    "sidebartrimcolor": "#FEFEFE",
-
-    "sectionbgcolor": "#FEFEFE",
-    "sectiontrimcolor": "#FEFEFE",
-    "sectiontextcolor": "#444444",
-
-    "relbarbgcolor": "#FEFEFE",
-    "relbartextcolor": "#444444",
-    "relbarlinkcolor": "#444444",
-
-    "bgcolor": "#FEFEFE",
-    "textcolor": "#444444",
-    "linkcolor": "#060",  # First Green
-
-    "headtextcolor": "#444444",
-    "headlinkcolor": "#444444",
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+import logging
 
-    # "codebgcolor"
-    # "codetextcolor"
-    "codetrimcolor": "#060",  # First Green
+# moksha logs warnings if qpid is missing, but we don't care
+logging.getLogger('moksha.hub').setLevel(logging.ERROR)
 
-    "footerbgcolor": "#FEFEFE",
 
-}
+# -- General configuration ------------------------------------------------
 
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
 
 # Add any Sphinx extension module names here, as strings. They can be
-# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc']
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.doctest',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.coverage',
+    'sphinx.ext.viewcode',
+    'sphinx.ext.napoleon',
+]
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
 
-# The suffix of source filenames.
-source_suffix = '.rst'
-
-# The encoding of source files.
-# source_encoding = 'utf-8'
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+source_suffix = ['.rst', '.md']
```

Comment:
Whoah I didn't know Sphinx could do markdown. That tempts me to convert my docs to markdown.